pkg/webhook/pod/util.go

```diff
@@ -56,9 +56,8 @@ func IsStatefulSetUpgrading(set *v1.StatefulSet) bool {
 // pd pod who would be deleted by statefulset controller
 // we add annotations to this pvc and delete it when we scale out the pd replicas
 // for the new pd pod need new pvc
-func addDeferDeletingToPVC(memberType v1alpha1.MemberType, podAC *PodAdmissionControl, tc *v1alpha1.TidbCluster, setName, namespace string, ordinal int32) error {
-	pvcName := operatorUtils.OrdinalPVCName(memberType, setName, ordinal)
-	pvc, err := podAC.pvcControl.GetPVC(pvcName, namespace)
+func addDeferDeletingToPVC(pvcName, namespace string, pvcControl controller.PVCControlInterface, tc *v1alpha1.TidbCluster) error {
```

Comment:
Can the `namespace` be get from `tc.Namespace`?

Reply:
updated.